modules/fetch/src/endpoints/advisory.rs

```diff
@@ -244,4 +272,47 @@ mod test {
 
         Ok(())
     }
+
+    #[test_context(TrustifyContext, skip_teardown)]
+    #[test(actix_web::test)]
+    async fn search_advisories(ctx: TrustifyContext) -> Result<(), anyhow::Error> {
+        use crate::endpoints::configure;
+        use actix_web::test::init_service;
+        use actix_web::web::Bytes;
+        use trustify_module_storage::service::fs::FileSystemBackend;
+
+        let db = ctx.db;
+        let graph = Graph::new(db.clone());
+        let (storage, _) = FileSystemBackend::for_test().await?;
+        let ingestor = IngestorService::new(graph, storage);
+        let app = init_service(App::new().configure(|mut config| configure(config, db))).await;
+        let mut response: PaginatedResults<AdvisorySummary>;
+
+        // No results before ingestion
+        let result = query(&app, "").await;
+        assert_eq!(result.total, 0);
+
+        // ingest some advisories
+        let data = include_bytes!("../../../../etc/test-data/mitre/CVE-2024-27088.json");
+        let id = ingest(&ingestor, data).await;
```

Comment:
Question: Did you do something wrt cargo manifest dir to simplify those ../.. before?

Reply:
Yes, but we weren't using `include_bytes!` then, and it will resolve paths relative to the file it's used in.